numpy-stubs/__init__.pyi

```diff
@@ -246,17 +250,23 @@ class _ArrayOrScalarCommon(
 _BufferType = Union[ndarray, bytes, bytearray, memoryview]
 
 class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
-    real: ndarray
-    imag: ndarray
+    @property
+    def real(self: _ArraySelf) -> _ArraySelf: ...
+    @property
```

Comment:
Shouldn't this be `@real.setter`?

Reply:
Oh wow. Yes, yes it absolutely should be.